Common/src/CConfig.cpp

```diff
@@ -2493,12 +2490,14 @@ void CConfig::SetConfig_Options() {
   /* DESCRIPTION: CHT interface coupling methods */
   /*  Options: NO, YES \ingroup Config */
   addEnumOption("CHT_COUPLING_METHOD", Kind_CHT_Coupling, CHT_Coupling_Map, DIRECT_TEMPERATURE_ROBIN_HEATFLUX);
-
-  /* DESCRIPTION: Thermal diffusivity constant */
-  addDoubleOption("THERMAL_DIFFUSIVITY", Thermal_Diffusivity, 1.172E-5);
-
   /* DESCRIPTION: Thermal diffusivity constant */
-  addDoubleOption("THERMAL_DIFFUSIVITY_SOLID", Thermal_Diffusivity_Solid, 1.172E-5);
+  addDoubleOption("SOLID_THERMAL_DIFFUSIVITY", Thermal_Diffusivity_Solid, 1.172E-5);
+  /* DESCRIPTION: Thermal conductivity used for heat equation */
+  addDoubleOption("SOLID_THERMAL_CONDUCTIVITY", Thermal_Conductivity_Solid, 0.0);
+  /* DESCRIPTION: Solids temperature at freestream conditions */
```

Comment:
Add the deprecated option to CConfig::SetConfig_Parsing error handling part.

Is it weird to have both diffusivity and conductivity?

Reply:
Ok!

It is weird to have both. For the heat solver, the only parameter that goes into computing the heat fluxes is thermal diffusivity, though one is used to specifiy the (in this case) redundant triple conductivity/density/capacity. I'm very open to remove them and leave a hint in the config template how to calculate diffusivity from them...

Reply:
Hmmm according to line 169 of the HeatSolver conductivity is used in some cases.
And used to set the diffusivity in the config :confused: 
I would go for a non-ambiguous way of specifying this. Hopefully consistent with the scalar solvers but also semi-backwards compatible (i.e. different option name but same quantity that needs to be specified by users).

Reply:
Yes but just to translate it to diffusivity. I'll go for `THERMAL_DIFFUSIVITY`, this way we can remove three ugly `SOLID_*` options :)

Reply:
Yes that's what I was saying. For the heat solver, one can either specify thermal conductivity alongside density and specific heat, or one goes for setting the diffusivity directly.
However when prescribing heat fluxes (as BC or in CHT), they have to be non-dimensionalized by rho*c_p, so it's maybe a better option to remove `SOLID_THERMAL_DIFFUSIVITY`as a config file option.

Reply:
Now that there's a energy equation in the incompressible solver, I could also remove the ungly `SOLID_*` options as the heat equation solver is active for pure solids only; using the standard `Kt_Constant` and so on.

Reply:
That (boundary fluxes) is a good point to raise in #1223, the wall boundaries are not implemented in the scalar solvers, only inlets and outlets (I think...).